app/models/user.rb

```diff
@@ -3,6 +3,7 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
 
   include Searchable
+
```

Comment:
Empty line not necessary, pain attention on unnecessary modifications. It can mess up the code.